src/index.html

```diff
@@ -1,19 +1,239 @@
 <!doctype html>
 <html lang="en">
-<head>
-  <meta charset="UTF-8">
-  <meta name="viewport"
-        content="width=device-width,
-          user-scalable=no,
-          initial-scale=1.0,
-          maximum-scale=1.0,
-          minimum-scale=1.0">
-  <meta http-equiv="X-UA-Compatible" content="ie=edge">
-  <title>Document</title>
-  <link rel="stylesheet" href="./style.css">
-</head>
-<body>
+  <head>
+    <meta charset="UTF-8">
+    <meta name="viewport"
+          content="width=device-width,
+            user-scalable=no,
+            initial-scale=1.0,
+            maximum-scale=1.0,
+            minimum-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <title>Document</title>
+    <link rel="stylesheet" href="./style.css">
+    <link href="https://fonts.googleapis.com/css?family=Roboto:400,500,700&display=swap" rel="stylesheet">
+  </head>
+  <body>
+    <header class="header">
+        <a >
+            <img src="images/logo.png" class="header-logo" alt="" width="40px" height="40px">
+        </a>
+      <nav class="site-navigation">
+        <ul class="site-navigation-list">
+          <li><a href="#" class="site-navigation-item is-active">Apple</a></li>
```

Comment:
nested tags must be placed on the new line always

Reply:
nested tags  on new line